uvicorn/middleware/wsgi.py

```diff
@@ -75,6 +76,11 @@ def build_environ(
 
 class WSGIMiddleware:
```

Comment:
```suggestion
class _WSGIMiddleware:
```
Doing this, we can improve how our tests looks like, and also make the import below more meaningful.